src/azure-cli-testsdk/azure/cli/testsdk/checkers.py

```diff
@@ -39,6 +39,24 @@ def __call__(self, execution_result):
                                               execution_result.output)
 
 
+class JMESPathCheckGreaterThan(object):  # pylint: disable=too-few-public-methods
```

Comment:
Not really. This allows for filtering lists by a predicate, so you apply a predicate to a list to get a smaller list of the same type. I don't see how to apply a predicate to a value to get boolean result.